app/templates/profile.hbs

```diff
@@ -0,0 +1,3 @@
+{{outlet}}
+  {{forms/profile-page}}
```

Comment:
Call the component, `forms/user-profile-form` and change all reference everywhere.